### PR TITLE
Rename archive file name

### LINKED ISSUE #1397: Rename the archive directory to past programs
**Is your feature request related to a problem? Please describe.**
- We need to change the archive as past programs

**Describe the solution you'd like**
- Rename this archive https://github.com/sef-global/sef-site/tree/master/scholarx/archive directory to `past-programs`
- Update all the links that referring that directory as well

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by Tharindu37**
## Purpose
Rename archive file name
The purpose of this PR is to fix #1397

## Goals
Rename archive file name and update all URL

## Approach
Rename archive file name

Preview Link
https://pr-1397-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: onelive/index.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h1 class="display-3 text-center mb-5">Take a look at our past OneLives</h1>
             </div>
             <div class="row justify-content-center mb-md-5">
                 <div class="col-9 text-center">
-                    <a href="archive/" class="btn btn-link border-0 btn-info">SEE MORE</a>
+                    <a href="past-programs/" class="btn btn-link border-0 btn-info">SEE MORE</a>
```
Comment (Piumal1999) [inline onelive/index.html]: I think we should keep the onelive archive as it is. Let's only change the scholarx archives.

File: onelive/index.html [hunk 2]
```diff
@@ -384,7 +384,7 @@ <h6 class="font-weight-bold">{{snippet.title}}</h6>
             </div>
             <a class="btn btn-default position-absolute hover-element"
                style="right: 20px; bottom: 20px"
-               href="/onelive/archive/?videoId={{snippet.resourceId.videoId}}">
+               href="/onelive/past-programs/?videoId={{snippet.resourceId.videoId}}">
```
Comment (anjula-sack) [inline onelive/index.html]: This shouldn't be changed. Keep it the way it was. @Tharindu37

File: scholarx/past-programs/2021-summer/index.html [hunk 3]
```diff
@@ -504,7 +504,7 @@ <h1 class="mb-0 text-onelive">Take a look at our past</h1>
         </div>
         <div class="row justify-content-center">
             <div class="col-9 text-center">
-                <a href="../../archive"
+                <a href="../../past-programs"
                    target="_self" class="btn btn-default">Check out Our Archive</a>
```
Comment (anjula-sack) [inline scholarx/past-programs/2021-summer/index.html]: Let's change this button to `Past Programs`

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Thank you! @Tharindu37

**Pull request merged**